subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,12 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files) found in inputs and outputs of tasks are handled gracefully from now on: as if they don't exist.
+
+For example copying into a directory with a leftover named pipe or broken symbolic link won't hang the build anymore.
```

Comment:
```suggestion
For example copying into a directory with a leftover named pipe or broken symbolic link won't break the build anymore.
```

The build hanged with a named pipe and failed with a symbolic link. I would go for `break`.